wire/internal/wire/wire_test.go

```diff
@@ -88,7 +88,14 @@ func TestWire(t *testing.T) {
 				t.Fatal(err)
 			}
 			wd := filepath.Join(gopath, "src", "example.com")
-			gen, errs := Generate(ctx, wd, append(os.Environ(), "GOPATH="+gopath), test.pkg)
+			gens, errs := Generate(ctx, wd, append(os.Environ(), "GOPATH="+gopath), []string{test.pkg})
+			if len(gens) > 1 {
```

Comment:
It could be interesting to permit this (for testing multiple-package generation or #505), but we should file an issue separately.

Reply:
Ack.